doc/requirements.txt

```diff
@@ -3,16 +3,16 @@ ipykernel
 sphinx==2.2.2
 m2r
 networkx>=2.0
-numpy>=1.17.4
+numpy>=1.16.3
 plotly
-quantum-blackbird>=0.2.3
-scipy==1.4.1
+quantum-blackbird
+scipy>=1.0.0
```

Comment:
Hey @ilan-tz --- how come these were changed?

Reply:
Hey - changed them to see if it somehow fixes the error in the readthedocs check but it didn't, so I reverted the commit!

Reply:
But I did notice that the requirements in \docs and in the root directory are different from each other, and also different from those in setup.py